middleware/middleware.js

```diff
@@ -77,4 +77,10 @@ function middleware (state, request, response, next) {
         response.end(error.toString())
       })
   })
+
+  // refiring data and end event if request.body exists since they won't fire on their own
+  if (request.body) {
+    request.emit('data', Buffer.from(JSON.stringify(request.body)))
+    request.emit('end')
+  }
```

Comment:
I wonder, is there a need to emit the events, why not just call `verifyAndReceive` directly? 

```js
  if (request.body) {
    verifyAndReceive(state, {
      id: id,
      name: eventName,
      payload: request.body,
      signature
    })

      .then(() => {
        response.end('ok\n')
      })

      .catch(error => {
        response.statusCode = error.status || 500
        response.end(error.toString())
      })
  }
```

Reply:
See my change at https://github.com/octokit/webhooks.js/pull/23/commits/b380f87c53e51cc9164f309e76becca4bd5cf16b which omits re-emitting the "data" event on the `request` object. Test is still passing, so unless anyone objects I’ll go ahead with that :)